packages/data-broker/src/__tests__/services/dhis/DhisService.test.ts

```diff
@@ -248,108 +248,176 @@ describe('DhisService', () => {
     });
   });
 
-  describe('pull()', () => {
-    describe('pull functionality', () => {
-      const basicEventOptions = {
+  describe('pullAnalytics()', () => {
```

Comment:
Duplicating shared test cases between `pullAnalytics` and `pullEvents` and restructuring test cases a bit